DEV/alter_crops_obic.R

```diff
@@ -232,11 +233,30 @@ cr <- cr[,V1 := NULL]
     
     # unknown: maggi, niger, roodzwenk, rijst, spurrie, seradelle, zonnekroon
    
-    # update the csv and Rdata file
-    # crops.obic <- copy(cr)
-    # fwrite(crops.obic, 'dev/crops_obic.csv')
-    # save(crops.obic, file = 'data/crops_obic.RData')
-    
+  # Add nitrogen use normes (stikstofgebruiksnormen)
+    nfcols <- c('nf_clay', 'nf_sand.other', 'nf_sand.south', 'nf_loess', 'nf_peat')
+  
```

Comment:
please set here a variable cols = c('nf_clay',.. and so on)
and then below
cr[grepl(xx,crop_name), c(cols) := list (0,0,0,0)]
this to avoid repetition